<ui version="4.0" >
 <class>devicesyncview_base</class>
 <widget class="QWidget" name="devicesyncview_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>kapp4_base</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <widget class="QDockWidget" name="dockWidget" >
     <property name="windowTitle" >
      <string>Connected Devices</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_2" >
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QListWidget" name="connectedDevicesWidget" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" >
         <item>
          <widget class="KComboBox" name="leftDeviceBox" />
         </item>
         <item>
          <widget class="QTreeView" name="leftTreeView" >
           <property name="contextMenuPolicy" >
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="moveRightButton" />
         </item>
         <item>
          <widget class="KPushButton" name="moveLeftButton" />
         </item>
         <item>
          <spacer name="verticalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <item>
          <widget class="KComboBox" name="rightDeviceBox" />
         </item>
         <item>
          <widget class="QTreeView" name="rightTreeView" >
           <property name="contextMenuPolicy" >
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QDockWidget" name="queueWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="windowTitle" >
       <string>Queue</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QListWidget" name="listWidget" >
          <property name="contextMenuPolicy" >
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
